examples/tracing-grpc/Cargo.toml

```diff
@@ -19,7 +19,7 @@ opentelemetry_sdk = { path = "../../opentelemetry-sdk", features = ["rt-tokio"]
 opentelemetry-stdout = { workspace = true, features = ["trace"] }
 prost = { workspace = true }
 tokio = { workspace = true, features = ["full"] }
-tonic = { workspace = true, features = ["server"] }
+tonic = { workspace = true, default-features = false, features = ["server"] }
```

Comment:
nit - this would conflict with #2710 based on whichever is merged first.

Reply:
commented, may be these can be made in sync. "server" seems conflicts on deeper "tonic" level and not fix possible without tonic patch.